R/print_method.R

```diff
@@ -30,11 +31,12 @@ tbl_format_header.tidySingleCellExperiment <- function(x, setup, ...) {
             align(paste0(names2(named_header), ":"), space=NBSP),
             " ", named_header) %>%
             # Add further info single-cell
-            append(sprintf(
-                "\033[90m Features=%s | Cells=%s | Assays=%s\033[39m",
-                number_of_features, nrow(x), 
-                paste(assay_names, collapse=", ")
-            ), after=1)
+          append(sprintf(
+              "\033[90m Features=%s | Cells=%s | Assays=%s | altExpNames=%s\033[39m",
```

Comment:
Thanks. 

I bit more elegant would be

- Replace `altExpNames` with `Assays_alternative`
- Conditionally add "| altExpNames=%s" if alternative experiment exists

Reply:
Fair point. I had considered adding the information on a conditional basis but personally I prefer to know even if it's empty. The standard `SCE` print method outputs `altExpNames(0):` when empty and `altExpNames(1): Antibody Capture` when it is populated (`Antibody Capture` being the name of the alternative experiment and can be any string). If you feel you would rather keep it only when it is non-NULL I can do that quite easily. 
In relation to the naming, it actually refers to the name of the experiment rather than the names of the assays, i.e. if it exists you will still have `counts` and `normcounts` or other as the assay names within the `altExp` slot. How about saying `Other_experiments=`?
I will also add the unit tests and other missing bits whilst I am at it. 

Thanks.

Reply:
I see.

OK the goal here is to provide an interface that promotes modularity. 

let's try to generalise the concept of experiment  and assay, still hinting to the non-modular nature of singlecellexperiment.

So I am in favour of printing the experiment within the object, if are 1,2,3 independently whether they base or alternative.

having said that,  there are a lot of ways to go about that. One possibiliy is to do

`Assays= base:counts,logcounts; Antibody capture:counts,logcounts; ADT:counts`

Other ideas are welcome!

Reply:
@Biomiha, I realised that the sentence "OK, the goal here is to provide an interface that promotes modularity. " sounded very bad in writing. I used a poor form to give the broad indication that "the tidy transcriptomics thrive on giving modularity (which sometimes diverges from the Bioc framework)."

I hope you did not take it personally!

Reply:
No dude, absolutely not. I work in biotech 😄 .
I agree with what you've said. Am working on implementation, just been very busy lately.